app/src/Class/EmailSender.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace src\Class;
+require 'vendor/autoload.php'; 
+
+use src\Interface\EmailSenderInterface;
+use PHPMailer\PHPMailer\PHPMailer;
+use PHPMailer\PHPMailer\Exception;
+
+class EmailSender implements EmailSenderInterface
+{
+    public $mail;
+    public function __construct($host, $port, $from) 
+    {
+        $this->mail = new PHPMailer;
+        $this->mail->isSMTP();                    
+        $this->mail->Host = $host;
```

Comment:
spaces between properties and function calls

Reply:
done